pkg/fmmon/alert.go

```diff
@@ -0,0 +1,36 @@
+package fmmon
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+// Alert sends the specified message to the specified webhook URL.
+func Alert(msg string, webhookURL string) {
```

Comment:
This function sits in a non-ideal position in the design space:

1. Its contract is too general to be easily understood (allegedly, this delivers _some kind_ of message to _some kind_ of webhook).
2. Its implementation is too specific to be used for many use cases (it's very clearly tied to Slack-compatible webhook APIs).
3. It's too simple to be worth testing (and probably too simple to be worth abstracting).